modules/mobile/spec/request/claims_and_appeals_overview_request_spec.rb

```diff
@@ -277,26 +277,11 @@
       end
     end
 
-    context 'when user is not authorized to use lighthouse or evss' do
-      before { iam_sign_in(user) }
-
-      context 'and is not authorized to use appeals' do
-        let(:user) { FactoryBot.build(:iam_user, :no_participant_id, :loa2) }
-
-        it 'returns 403 status' do
-          VCR.use_cassette(good_claims_response_vcr_path) do
-            VCR.use_cassette('mobile/appeals/appeals') do
-              get('/mobile/v0/claims-and-appeals-overview', headers: iam_headers, params:)
-              expect(response).to have_http_status(:forbidden)
-            end
-          end
-        end
-      end
-
-      context 'but is authorized to use appeals' do
-        let(:user) { FactoryBot.build(:iam_user, :no_participant_id) }
+    context 'when user has access to only claims service' do
+      before { allow_any_instance_of(IAMUser).to receive(:ssn).and_return(nil) }
 
-        it 'and appeals service succeed' do
+      if lighthouse_flag
```

Comment:
should this be able to pass on both evss and lighthouse?

Reply:
No because user can't access evss if they dont have both `loa3` & `ssn` as well (which is required for appeals)

Reply:
Could we instead set loa3 to nil so it'll work on both lighthouse and evss?

Reply:
@aherzberg  It looks like loa3 can't be set to nil.. but loa can. But setting loa to nil still allowed access to appeals

Reply:
@aherzberg fixed